npm/parseSelector_test.go

```diff
@@ -513,6 +513,10 @@ func TestFlattenNameSpaceSelector(t *testing.T) {
 		t.Errorf("TestFlattenNameSpaceSelector failed @ 1st selector length check %+v", testSelectors)
 	}
 
+	if !reflect.DeepEqual(testSelectors[0], *firstSelector) {
```

Comment:
There are multiple time to check length of returned values from FlattenNameSpaceSelector and deepEqual in this function and TestFlattenNameSpaceSelectorWoMatchLabels function. So, I think it is better to make one helper function to check them together.

Reply:
Makes sense.